packages/analytics/matomo/index.tsx

```diff
@@ -11,6 +15,14 @@ export function trackMatomoPageView(url: string) {
    matomoPush(['trackPageView']);
 }
 
+export function trackPiwikPreferredStore() {
+   if (typeof window !== 'undefined') {
+      const host = getShopifyStoreDomainFromCurrentURL();
```

Comment:
Isn't the react-commerce store US only? I think we can just hardcode this to `ifixit-us` instead of parsing the domain.

Reply:
yes rn react-commerce is only for the us, but we would need to handle ifixit-test still so we cant just straight hardcode it

Reply:
I'm fine with leaving it as is then. No sense in simplifying to just US and the test stores only to have to expand it in the future.